praw/helpers.py

```diff
@@ -111,6 +112,88 @@ def valid_redditors(redditors, sub):
             if resp['ok']]
 
 
+def all_submissions(reddit_session, subreddit, highest_timestamp=None, lowest_timestamp=0, verbosity=1):
+    """Yield submissions between ``highest_timestamp`` and ``lowest_timestamp``
```

Comment:
For the sake of (theoretically?) lowering required time and resources, you may want to make the lower bound be `None` as the default in the kwarg, and if it happens to be None, make it the time of the subreddit's creation (from the subreddit object provided or, retrieved if it is a string).